### PR TITLE
Feature/AT-8960-Add Acquisition Package ID To Contracts Tables

### PR DESCRIPTION
**Pull request by DGabriel-CCPO**
- Adds column `Acquistion Package` to `Contract Considerations` table
- Adds column `Acquistion Package` to `Contract Type` table

Ticket: AT-8960

### CODE REVIEW

**Comment by jasonburkert-ccpo** [comment]
Since SNOW PR's are so inscrutable, I recommend being explicit about what's changed in the pull request.  Over time you get used to parsing the XML visually, but until then it's good to include screenshots and describe just what was changed.

For example:

### New column in `Contract Considerations` table

image

### New column in `Contract Type` table

image

File: f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?><record_update>
     <sys_dictionary action="INSERT_OR_UPDATE" element="current_contract_and_recurring_information" table="x_g_dis_atat_acquisition_package">
-        <active>false</active>
+        <active>true</active>
```
Comment (jasonburkert-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml]: @DGabriel-CCPO What happened here?  How did the Current Contract Information table change?

Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml]: Looks like the entire table was made active, but wasn't it already active?

**Review by ericyoungquist-ccpo — DISMISSED** [review]

File: f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml (hunk 1 above)
Comment (DGabriel-CCPO) [reply f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml]: @jasonburkert-ccpo For some reason when I checked the able on my branch it wasn't active but i didnt disable and figured it must have been set to false in on of the recent changes

Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml]: Update: This is frustrating when looking at just the XML

This is column `Acquisition Package.Current Contract Information` that references the child table `Current Contract Information`.  @tomarnold-ccpo Disabled this recently with the expectation that data migration would be performed post-release by executing Action `MIGRATION: Contract History`

**Comment by DGabriel-CCPO** [comment]
Screenshot 2023-05-17 at 9 41 44 AM
unchecked the active box for the reference column current contract information in the acquisition package table.

File: f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml (hunk 1 above)
Comment (jasonburkert-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_acquisition_package_current_contract_and_recurring_information.xml]: After some testing re: data migration this was set to `true` in #383

**Review by jasonburkert-ccpo — APPROVED** [review]

**Pull request merged**